docs/source/governance.rst

```diff
@@ -8,10 +8,16 @@ Core Maintainers
 - William Falcon (`williamFalcon <https://github.com/williamFalcon>`_) (Lightning founder)
 - Jirka Borovec (`Borda <https://github.com/Borda>`_)
 - Ota Jašek (`otaj <https://github.com/otaj>`_)
-- Akihiro Nitta (`akihironitta <https://github.com/akihironitta>`_)
+
+Core Contributors
+-----------------
+- Atharva Phatak (`Atharva-Phatak <https://github.com/Atharva-Phatak>`_)
+- Shion Matsumoto (`matsumotosan <https://github.com/matsumotosan>`_)
+- JongMok Lee (`lijm1358 <https://github.com/lijm1358>`_)
 
 Alumni
 ------
-- Teddy Koker (`teddykoker <https://github.com/teddykoker>`_)
-- Annika Brundyn (`annikabrundyn <https://github.com/annikabrundyn>`_)
+- Akihiro Nitta (`akihironitta <https://github.com/akihironitta>`_)
```

Comment:
@akihironitta are you fine with becoming alumni or will you stay around? :)

Reply:
Feel free to revert b9ac91e to drop me.

Reply:
Oh, we'd like to keep you around :zap: